### PR TITLE
minor changes updates on gush and on year of .formatter.yml

### PR DESCRIPTION
**Pull request by cordoval**
|Q            |A  |
|---          |---|
|Bug Fix?     |no |
|New Feature? |no |
|BC Breaks?   |no |
|Deprecations?|no |
|Tests Pass?  |no |
|Fixed Tickets|   |
|License      |MIT|

### CODE REVIEW

File: .formatter.yml [hunk 1]
```diff
@@ -10,7 +10,7 @@ header: |
     /*
      * This file is part of the Elcodi package.
      *
-     * Copyright (c) 2014 Elcodi.com
+     * Copyright (c) 2015 Elcodi.com
```
Comment (mmoreram) [inline .formatter.yml]: Great, and the year is not `2015`, but `2014-2015`

**Comment by mmoreram** [comment]
Great :) Could you please squash to be merged? :)

**Comment by cordoval** [comment]
@mmoreram

**Comment by mmoreram** [comment]
Thanks @cordoval :D

**Pull request merged**